lang/es.json

```diff
@@ -26,6 +26,8 @@
   "black": "negro",
   "personal-details": "Detalles personales",
   "skills": "Habilidades",
+  "upload-profile-image": "Subir imagen de perfil",
+  "clear-profile-image": "Claro",
```

Comment:
For `clear-profile-image`, you could check `clear-settings` to get the right translated word for clear as I think it applies to all languages.

Reply:
thanks for pointing out this, I used chatGPT to translate the words but to Clear I think it chose the meaning of 'understood' instead of 'remove', hahah

I updated the translation again in a new commit, please check